test/dotnet-add-p2p.Tests/GivenDotnetAddP2P.cs

```diff
@@ -626,5 +642,79 @@ public void WhenReferenceIsRelativeAndProjectIsNotInCurrentDirectoryAndForceSwit
             csproj.NumberOfItemGroupsWithoutCondition().Should().Be(noCondBefore + 1);
             csproj.NumberOfProjectReferencesWithIncludeContaining(setup.ValidRefCsprojRelPath.Replace('/', '\\')).Should().Be(1);
         }
+
+        [Fact]
+        public void ItCanAddRefWithCondOnCompatibleFramework()
```

Comment:
Let's expand this test name. ItCanAddReferenceWithConditionOnCompatibleFramework. Much nicer to read.